frontend/lib/features/widget/video_list_item.widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
```

Comment:
See comment about audio player. Also this no requirement of the audio player ticket and should be removed either way